engine/class_modules/paladin/sc_paladin_holy.cpp

```diff
@@ -648,15 +648,35 @@ void paladin_t::generate_action_prio_list_holy_dps()
   cds->add_action( "use_items,if=(buff.avenging_wrath.up)" );
   cds->add_action( "seraphim" );
 
-  priority->add_action( this, "Shield of the Righteous" );
+  priority->add_action( "restore_mana,if=mana<10000", "Assume infinite mana." );
```

Comment:
Hi, I wrote the logic behind these changes. You could definetly make this an advanced option,  however I don't know if there's anyone out there actually trying to optimize their hpal dps per unit mana. It's just a matter of adding or dropping haste.

Reply:
removed this line, you should be able to add or remove haste to see that behavior instead

Reply:
In all reality the only time simc is amazingly accurate for healers is when you are going for that 100% ignore healing and focus on dps playstyle
You can still get critical information but in the grand scheme of things when you ignore mana you don't show a true paladin dps. It would be like ignoring mana for arcane mages

But thanks for removing it :D